lib/orderbook/OrderBook.ts

```diff
@@ -278,45 +289,68 @@ class OrderBook extends EventEmitter {
       this.logger.debug(`placeOrder max time exceeded. order (${JSON.stringify(order)}) won't be fully matched`);
 
       // returning the remaining order to be rolled back and handled by the initial call
-      return Promise.resolve({
+      return {
         internalMatches: [],
         swapResults: [],
         remainingOrder: order,
-      });
+      };
     }
 
-    // perform match. maker orders will be removed from the repository
+    // perform matching routine. maker orders that are matched will be removed from the order book.
     const tp = this.getTradingPair(order.pairId);
     const matchingResult = tp.match(order);
 
-    // instantiate the final response object
-    const result: PlaceOrderResult = {
-      internalMatches: [],
-      swapResults: [],
-      remainingOrder: matchingResult.remainingOrder,
+    /** Any portion of the placed order that could not be swapped or matched internally. */
+    let { remainingOrder } = matchingResult;
+    /** Local orders that matched with the placed order. */
+    const internalMatches: OwnOrder[] = [];
+    /** Successful swaps performed for the placed order. */
+    const swapResults: SwapResult[] = [];
```

Comment:
This should be changed to `swapSuccesses` (also in `PlaceOrderResult`).

Reply:
I thought about doing this but I wasn't sure about it, but if you think it should be changed I'll go ahead. I may also change the name of the `SwapResult` type.

Reply:
I went ahead and renamed all references I could find to make it consistent.